shared/people/follow-notification/index.stories.js

```diff
@@ -5,11 +5,15 @@ import * as Sb from '../../stories/storybook'
 import FollowNotification, {type Props} from '.'
 import moment from 'moment'
 
+// Mock moment.now(), since moment uses this for everything
+// $ForceType
+moment.now = () => 1538066670959
```

Comment:
we shouldn't do it this way. we already have mocks for storybook, see the timestamp mocks etc